src/sql/workbench/parts/notebook/electron-browser/outputs/notebookMarkdown.ts

```diff
@@ -197,15 +198,17 @@ export class NotebookMarkdownRenderer {
 			// ignore
 		}
 		let originIndependentUrl = /^$|^[a-z][a-z0-9+.-]*:|^[?#]/i;
-		if (base && !originIndependentUrl.test(href) && !fs.existsSync(href)) {
-			href = this.resolveUrl(base, href);
-		}
-		try {
-			href = encodeURI(href).replace(/%25/g, '%');
-		} catch (e) {
-			return null;
-		}
-		return href;
+		return this.fileService.exists(URI.file(href)).then(exists => {
+			if (base && !originIndependentUrl.test(href) && !exists) {
```

Comment:
As discussed this needs to return the href itself. 

I'd suggest adding a return type to this method so that mistake isn't made again in the future. The return type should just be string | undefined

Reply:
I removed the file exists check and alternatively am checking if the path is absolute since we call the resolve url only for relative paths.